src/features/external-simulation/providers/ExternalSimulation.js

```diff
@@ -0,0 +1,43 @@
+import io from 'socket.io-client';
+
+export class ExternalSimulation {
+
+    constructor (simulationManager) {
+        this.simulationManager = simulationManager;
+        this.simulators = [];
+        this.socket = io('http://localhost:3000/', {
+            'path': '/gateway',
+        });
+        this.registerHandlers();
+    }
+
+    registerHandlers () {
+        this.socket.on('connect', () => {
```

Comment:
Why is this not in renew-simulator-client?

Reply:
I initially discarded the renew-simulator-client and decided to use the socket directly in the external-simulation module. Another abstraction might be useful however for other people planning JavaScript projects using the renew-simulator-gateway. I'll think about it.